docs/core/rid-catalog.md

```diff
@@ -107,19 +114,33 @@ wish to check if new ones are added, please check back here.
     * `ubuntu.14.04-x64`
     * `ubuntu.14.10-x64`
     * `ubuntu.15.04-x64`
+    * `ubuntu.15.10-x64`
+    * `ubuntu.16.04-x64`
+    * `ubuntu.16.10-x64`
 * CentOS
     * `centos.7-x64`
-    * `centos.7.1-x64`
 * Debian
     * `debian.8-x64`
-    * `debian.8.2-x64`
+* Fedora
+    * `fedora.23-x64`
+    * `fedora.24-x64`
+* OpenSUSE
+    * `opensuse.13.2-x64`
+    * `opensuse.42.1-x64`
+* Oracle Linux
+    * `ol.7-x64`
+    * `ol.7.0-x64`
+    * `ol.7.1-x64`
+    * `ol.7.2-x64`
 * Currently supported Ubuntu derivatives 
     * `linuxmint.17-x64`
     * `linuxmint.17.1-x64`
     * `linuxmint.17.2-x64`
     * `linuxmint.17.3-x64`
+    * `linuxmint.18-x64`
 
-## OS X RIDs
+## Mac RIDs
```

Comment:
what do you think @richlander? Should we keep OS X as you suggested back in that earlier comment? I was following how we're calling the OSes on this page for this change: https://www.microsoft.com/net/core